development/forms.py

```diff
@@ -1,11 +1,13 @@
+import re
+
+from auth_app.models import Bot
```

Comment:
Wrong import group. this is not a 3rd library it is a local import.

Reply:
Ok, I try the "sort imports" of VSCode, but it seems that it doesn't so well. So I going to dismiss this way to sort the imports. I going to sort manually.